sway-core/src/ir_generation/const_eval.rs

```diff
@@ -491,15 +548,15 @@ fn const_eval_typed_expr(
             then,
             r#else,
         } => {
-            match const_eval_typed_expr(lookup, known_consts, condition)? {
+            match dbg!(const_eval_typed_expr(lookup, known_consts, condition))? {
```

Comment:
Are those uses of `dbg!` leftovers or intended?

Reply:
Argh! Fixed.